python/setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+setup(
```

Comment:
You'll want to include any dependencies this package relies on here via `install_requires`. Here's an example: https://github.com/azavea/stacframes/blob/master/setup.py#L12

I see at least scipy, numpy, and parsec?